README.md

```diff
@@ -26,4 +34,6 @@
 
 ### Reference
 
+`make docserver` will start a `yard` server using the RDoc comments from the codebase. RDoc in this application is a work-in-progress and should improve over time. As of this writing, the index page generated contains broken links to our markdown documentation, but they "files" navigation displays them properly.
```

Comment:
I'd suggest we document the prerequisite of `gem install yard` here. Without that, the make command fails (for me, anyway).

Reply:
`bundle install` should cover that as it's in the bundle right?

Reply:
We could have `make install` be run prior to every run of `make docserver` but I thin that would get annoying.

I'll update the readme to know that prior to running for the first time, `bundle install` or `make install` needs to be run.

My initial reaction was "but that's always the case for gems", but after having sat with this for a bit I think it's worth noting explicitly for this as it likely isn't clear it's a bundled gem that is doing this work. Thanks for pointing it out.

Reply:
I could've sworn that I got that error *after* running `bundle install`, which is why I mentioned it. I just tried this, though:

1. Run `gem uninstall yard`.
2. Try `make docserver`: returns `rbenv: yard: command not found`, as expected.
3. Run `bundle install`.
4. Try `make docserver`: the yard server starts up successfully.

I was so baffled by this that I looked at my shell history from where I first checked out the branch:

```bash
10123  git pull
10124  git checkout yard-and-makefile
10125  bundle install
10126  make docserver (this failed with 'command not found')
10127  rm -rf .yardoc (tried clearing the cache as suggested, in case there was any)
10128  make docserver (failed again)
10129  gem install yard
10130  yard server --reload (working now)
```

After that, `make docserver` worked as well, as expected. So, I think it's still possible for yard not to work even if you *do* run `bundle install` first, but I'm not exactly sure why. Since I can't replicate it, I think I'm fine with the additional tip text as-is.

Reply:
Super weird. Thanks for confirming so we can keep an eye out for this in the future.